vllm/worker/model_runner.py

```diff
@@ -212,6 +212,26 @@ def load_model(self) -> None:
                            "but the KV cache data type is not FP8. "
                            "KV cache scaling factors will not be used.")
 
+    def save_model(self, path: str, max_size: int) -> None:
```

Comment:
Again this name is too general so it seems not proper to save the model in this way. We may call it `save_sharded_state_dicts` for now given its functionality.

Reply:
would probably make even more sense to make this a part of the `StateDictLoader`

Reply:
renamed to `save_sharded_state` and moved internal logic to `ShardedStateLoader`